specs/nonbibrecord.proto

```diff
@@ -2,27 +2,32 @@ syntax = "proto3";
 import "status.proto";
 package adsmsg;
 
-message NonBibRecord {
+message NonBibSolrRecord {
     string bibcode = 1;
-    int32 id = 2;
-    repeated string authors = 3;
-    bool refereed = 4;
-    repeated string simbad_objects = 5;
-    repeated string grants = 6;
-    repeated string citations = 7;
-    float boost = 8;
-    int32 citation_count = 9;
-    int32 read_count = 10;
-    int32 norm_cites = 11;
-    repeated string readers = 12;
-    repeated int32 downloads = 13;
-    repeated int32 reads = 14;
-    repeated string reference = 15;
-    repeated string ned_objects = 16;
-    Status status = 17;
+    int32 author_count = 2;
+    bool refereed = 3;
+    repeated string simbad_objects = 4;
+    repeated string grants = 5;
+    repeated string citations = 6;
+    float boost = 7;
+    int32 citation_count = 8;
+    int32 read_count = 9;
+    repeated string readers = 10;
+    repeated int32 downloads = 11;
+    repeated int32 reads = 12;
+    repeated string reference = 13;
+    repeated string ned_objects = 14;
+    // data field keys are link sub types, values are counts    
+    map<string, int32> data = 15;
```

Comment:
I thought we'll need the following values:

for solr:

`data: ['NED:15', 'CDS:5']`

for resolver service:

`data_links`: {'NED': ['http://one/twoo', 'http://foo.bar.com...']`

So, if that's correct, then we'll have multiple recipients inside the message (for solr, for resolver) - but in general, I'd like to avoid 'hardcoding' implementation into the name.

`NonBib` was kind of vague, `NonBibSolr` might be misleading - can we find a different name that encapsulates the nature of the held data?